### ISSUE TITLE
On some dataset，pre-trained weights result is not good

### ISSUE THREAD

**Issue by laigq2015** [SHI-Labs/VMFormer#7]
Hi, thank you very much for your great work!
I tested this project with several img from  VideoMatte240K test set, the result is not good。
I use the Inference & Evaluation command line with the pre-trained weights which download from google drive，I found `error in ms_deformable_im2col_cuda: invalid device function` error when runing inference_vm.py，but get result and the result the result is not good。
Is anything wrong, pre-trained weights or `error in ms_deformable_im2col_cuda: invalid device function`  cause result poor。

**Comment by chrisjuniorli** [comment]
Thanks for your interest in our work. I wonder have you met the same error when you run `make.sh` to install the CUDA version of MultiScaleDeformableAttention, and could you specify your environment, like pytorch version, cuda version, etc.?

**Comment by laigq2015** [comment]
Thanks for your reply。
Yes，I meet  error when I run `make.sh` to install the CUDA version of MultiScaleDeformableAttention。
My environment created by command `conda install pytorch torchvision cudatoolkit=10.2 -c pytorch`。

**Comment by laigq2015** [comment]
I have re-install environment  by command `conda install pytorch==1.9.0 torchvision==0.10.0 cudatoolkit=9.2 -c pytorch`，
the problem has been resolved，thanks a lot。

**Comment by negintabaraki** [comment]
Hi, I have a question regarding the data, how did you prepare the data for training?
